src/client/localplayer.h

```diff
@@ -141,6 +141,8 @@ class LocalPlayer : public Player
 
 	void setCollisionbox(const aabb3f &box) { m_collisionbox = box; }
 
+	aabb3f getCollisionbox() const { return m_collisionbox; } //TheTermos
```

Comment:
const aabb3f &

Reply:
Received convention ;) the other one as well, it was already like that elsewhere in the code, but I get the point.

I'm going to modify these, thanks, but this code will keep changing anyway, so I propose let's leave such touches for the review phase, I'll try to get it tidy by then.

Reply:
changed as requested